src/acceptance-testing/release-checklist.md

```diff
@@ -0,0 +1,33 @@
+# Release Readiness Checklist (RRC)
```

Comment:
Candidly, I seriously doubt that we're going to have all of this ready before each release. We have no automation to perform on-demand performance benchmarks - in fact, monitoring devnets is the main method by which we discover performance problems.

I'd rather this be a shorter list with clearer, more attainable criteria than a longer list with aspirational thing we won't get to. That way, folks are more likely to actually follow the process and feel like it adds value.

Reply:
Agreed. And others have provided similar feedback. The document has since been significantly simplified. More process-heavy, or aspirational, ideas have been captured in a "Future Improvements" section for future reconsideration.
Thanks!